test/src/local_storage/shared_prefs_storage_test.dart

```diff
@@ -0,0 +1,103 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:matomo_tracker/src/local_storage/shared_prefs_storage.dart';
+import 'package:mocktail/mocktail.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+import '../../ressources/mock/data.dart';
+
+class MockSharedPreferences extends Mock implements SharedPreferences {}
+
+void main() {
+  group('SharedPrefsStorage', () {
+    late MockSharedPreferences mockPrefs;
+    late SharedPrefsStorage sharedPrefsStorage;
+
+    setUpAll(() {
+      mockPrefs = MockSharedPreferences();
```

Comment:
Why not using `SharedPreferences.setMockInitialValues()` ? And use the real  `SharedPreferences` object instead of a mock ?

Reply:
Actually keeping the mock is better as I don't really care about the value in my shared prefs but I want to confirm that a specific method has been called properly, hence why I use `verify`.